common/v2/features/Dashboard/Dashboard.tsx

```diff
@@ -98,30 +98,6 @@ export default function Dashboard() {
                 </div>
               </div>
             </div>
-            <AddressMetadataContext.Consumer>
-              {({ readAddressMetadata }) => (
-                <TransactionContext.Consumer>
-                  {({ transactions }) => {
-                    return (
-                      <TransactionHistoryContext.Consumer>
-                        {({ transactionHistories }) => {
-                          return (
-                            <div className="Dashboard-desktop-bottom">
```

Comment:
Please remove SCSS class

Reply:
Funnily enough this one doesn't actually exist